### DISCUSSION TITLE
keybinds_hinst.sh launch/execute ever possible?

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1534] [Q&A] [answered]
Launching apps or executing keybindings over the `keybindings_hint.sh` doesn't work anymore, but does it ever worked i'm not sure so i'm asking before opening an issue.

**Comment by kRHYME7** [comment, marked as answer]
It still opens my firefox and code 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/fe9ba141-6543-41db-a7e7-4b6bf7fdce31)



also this is base on ` hyprctl dispatch ` plus the keybind so if this  broke might be your configs. (unless hyprland changes something again)

**Reply by dieBakterie** [reply]
OK seems it's Hyprland, Switched to General Hyprdots and doesn't work for me propably after a Reinstall again if not then git or i don't know

**Reply by kRHYME7** [reply]
other keybinds work?  also if you use the keybinds it opens?

**Reply by dieBakterie** [reply]
Keybinds Work but Open over Keybinds hint doesn't. No Matter If Double clicked, enter or what ever, it just closes and reopens the keybinds hint resetted Like when opened over the Module/keybind

**Comment by dieBakterie** [comment]
```
❯ keybinds_hint.sh
cut: Trenner muss ein einzelnes Zeichen sein
„cut --help“ liefert weitere Informationen.
cut: Trenner muss ein einzelnes Zeichen sein
„cut --help“ liefert weitere Informationen.
```
Output when trying in terminal entry is Launch Application launcher(rofi). Scripts are the stock ones from Hyprdots main branch.

**Reply by dieBakterie** [reply]
Same porblem and error with the script from hyprdots main branch.

**Reply by kRHYME7** [reply]
Thanks updated the PR now!


the error says ` cut ` cannot use `   ` delimeter  

Here's the fix


lines 384-386
```
sel_1=$(awk -F "${kb_hint_delim:->}"  '{print $1}' <<< "$selected" | awk '{$1=$1};1')
sel_2=$(awk -F "${kb_hint_delim:->}"  '{print $2}' <<< "$selected" | awk '{$1=$1};1')
run="$(grep "$sel_1" <<< "$metaData" | grep "$sel_2")"
```

**Reply by kRHYME7** [reply]
And works even with a flashlight 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/adfb3081-ab95-4ad8-959d-bf7790fd2810)

**Reply by dieBakterie** [reply]
Nice and xd
